src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/MethodeLinksBodyProcessor.java

```diff
@@ -175,14 +175,14 @@ private boolean isRemovable(final Node aTag) {
             return Collections.emptyList();
         }
 
-        URI documentsUri = UriBuilder.fromUri(uri).queryParam("uuid", tags.values().stream().distinct().toArray()).build();
         ClientResponse clientResponse = null;
         try {
-            clientResponse = documentStoreApiClient.resource(documentsUri)
+            clientResponse = documentStoreApiClient.resource(uri)
                     .accept(MediaType.APPLICATION_JSON_TYPE)
+                    .type(MediaType.APPLICATION_JSON_TYPE)
                     .header(TransactionIdUtils.TRANSACTION_ID_HEADER, transactionId)
                     .header("Host", "document-store-api")
-                    .get(ClientResponse.class);
+                    .post(ClientResponse.class, tags.values().stream().distinct().toArray());
```

Comment:
Could you please try if this works with just `tags.values()` which is already a `Collection<String>`. Distinct is anyways done in the `document-store-api` as well, it doesn't hurt here either so it's not important that you remove it or keep it. I'm focusing more on if we need Array, or why not List then? Is the payload serialized correctly.

Reply:
A List is serialized the same way so we can use just tags.value().